e2e-test-audit.csv

```diff
@@ -0,0 +1,292 @@
+backup snapshots schedule describe,,N,N,,,X,X,,
+backup snapshots schedule update,,N,N,,,X,X,,
+backups snapshots list,,N,N,,,X,X,,
+backup disable,,N,Y,,,X,X,,
+backup enable,,N,N,,,X,X,,
+backup restore start,,N,N,,X,X,X,,
+backup restore list,,N,N,,X,X,X,,
+backup checkpoint list,,N,N,,X,X,X,,
+cluster apply,,N,N,,,X,X,internal/cli/opsmanager/clusters,"e2e/cloud_manager/deploy_replica_set_test.go, e2e/cloud_manager/deploy_sharded_cluster_test.go"
+cluster create,,Y,Y,,,X,X,,
+cluster delete,,Y,Y,,,X,X,,
+cluster describe,,Y,Y,,,X,X,,
+cluster list,,Y,Y,,,X,X,,
+cluster restart,,N,N,,,X,X,,
+cluster shutdown,,Y,Y,,,X,X,,
+cluster startup,,N,N,,,X,X,,
+cluster unmanage,,Y,Y,,,X,X,,
+cluster update,,N,N,,,X,X,,
+cluster index create,,N,N,,,X,X,,
+dbuser create,,N,Y,,,X,X,internal/cli/opsmanager/dbusers,e2e/cloud_manager/dbusers_test.go
+dbuser delete,,N,Y,,,X,X,,
+dbuser list,,N,Y,,,X,X,,
+diagnose-archive download,,N,N,,,X,X,internal/cli/opsmanager/diagnosearchive,
+featurePolicy list,,Y,Y,,,X,X,internal/cli/opsmanager/featurepolicies,e2e/cloud_manager/feature_policies_test.go
+featurePolicy update,,Y,Y,,,X,X,,
+logs jobs collect,,N,N,,,X,X,internal/cli/opsmanager/logs,
+logs jobs download,,N,N,,,X,X,,
+logs jobs delete,,N,N,,,X,X,,
+logs jobs list,,N,N,,,X,X,,
+maintenanceWindows create,,Y,Y,,,X,X,internal/cli/opsmanager/maintenance,e2e/cloud_manager/maintenance_test.go
+maintenanceWindows delete,,Y,Y,,,X,X,,
+maintenanceWindows list,,Y,Y,,,X,X,,
+maintenanceWindows update,,Y,Y,,,X,X,,
+database describe,,N,N,,,X,X,internal/cli/opsmanager/metrics,
+database list,,N,N,,,X,X,,
+disk describe,,N,N,,,X,X,,
+disk list,,N,N,,,X,X,,
+process,,N,N,,,X,X,,
+monitoring enable,,Y,Y,,,X,X,internal/cli/opsmanager/monitoring,
+monitoring disable,,Y,Y,,,X,X,,
+monitoring stop,,Y,Y,,,X,X,,
+owner create,,Y,,,,X,,internal/cli/opsmanager/owner,
+process describe,,N,N,,,X,X,internal/cli/opsmanager/processes,
+process list ,,N,Y,,,X,X,,
+security enable,,N,N,,,X,X,internal/cli/opsmanager/security,
+server list,,Y,Y,,,X,X,internal/cli/opsmanager/servers,e2e/cloud_manager/servers_test.go
+softwareComponent list,,N,N,,,X,,internal/cli/opsmanager/softwarecomponents,
+versionManifest update,,N,,,,X,,internal/cli/opsmanager/versionmanifest,
+serverUsage orgs hosts list,,N,,,,X,,internal/cli/opsmanager/serverusage/projects/servertype,
+serverUsage orgs serverType get,,N,,,,X,,,
+serverUsage orgs serverType set,,N,,,,X,,,
+serverUsage project hosts list,,N,,,,X,,,
+serverUsage project serverType get,,N,,,,X,,,
+serverUsage project serverType set,,N,,,,X,,,
+serverUsage capture,,N,,,,X,,,
+serverUsage download,,N,,,,X,,,
+performanceAdvisor namespace list,,N,N,,X,X,X,internal/cli/performanceadvisor,
+performanceAdvisor slowQueryLogs list,,N,N,,X,X,X,,
+performanceAdvisor suggestedIndexes list,,N,N,,X,X,X,,
+,,,,,,,,,
+,,,,,,,,,
+,,,,,,,,,
+N,57,135,57,,,,,,
+Y,112,19,52,,,,,,
+Total,169,154,109,,,,,,
+% Coverage,66.27%,12.34%,47.71%,,,,,,
```

Comment:
similar to above, are contributors expected to calculate this when updating this file?

Reply:
Ideally yes, but they probably won’t